pkgs/by-name/lz/lzip/package.nix

```diff
@@ -7,12 +7,13 @@
 
 stdenv.mkDerivation rec {
   pname = "lzip";
-  version = "1.23";
+  version = "1.24";
   outputs = [ "out" "man" "info" ];
 
   src = fetchurl {
     url = "mirror://savannah/lzip/${pname}-${version}.tar.gz";
-    sha256 = "sha256-R5LAR93xXvKdVbqOaKGiHgy3aS2H7N9yBEGYZFgvKA0=";
+    sha256 = "d42659229b10e066eeb6e81eb673cdd893b672e512d26719c2d95975556ca56c";
+    # hash from release announcement
```

Comment:
IMHO, I don't think this comment is necessary here.

Reply:
> IMHO, I don't think this comment is necessary here.

I don't think this comment is necessary here.

Reply:
Hey hey nice one !

Reply:
Seriously though, having providence information is better than "nix build gave me this hash".

Reply:
both arguments are valid, let's reach a conclusion while being respectful and polite